### DISCUSSION TITLE
Downtime for https://search.laksith.dev

### DISCUSSION THREAD

**Discussion by laksith19** [searxng/searx-instances#399] [Report instance downtime]
My instance instance now has an unexpected lower CSP rating and unusually high response time. I'll be taking the instance offline at 1:00 AM Sep 10 UTC to debug this issue. Will post an update here when I bring it back up.

**Comment by laksith19** [comment]
Instance is back up!

**Comment by unixfox** [comment]
Thank you for the notification
